src/main/scala/utils/images/Image.scala

```diff
@@ -282,7 +282,7 @@ trait VectorizedImage extends Image {
 
   def vectorToImageCoords(v: Int): Coordinate
```

Comment:
Yeah, I think that's a fine idea. We haven't done this much yet, but given
that this thing is potentially "dangerous" I guess we should go ahead and
do this.

On Mon, Feb 22, 2016 at 1:00 PM, Shivaram Venkataraman <
notifications@github.com> wrote:

> In src/main/scala/utils/images/Image.scala
> <https://github.com/amplab/keystone/pull/233#discussion_r53691860>:
>
> > @@ -282,7 +282,7 @@ trait VectorizedImage extends Image {
> >
> >    def vectorToImageCoords(v: Int): Coordinate
>
> Can we make this method and the coord object private ? Right now coord is
> only used by this method, so it gives us some assurance it won't be
> accessed outside of it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amplab/keystone/pull/233/files#r53691860>.
>